src/Analyzers/CSharp/CodeFixes/ConvertToRecord/ConvertToRecordEngine.cs

```diff
@@ -264,8 +264,12 @@ await RefactorInitializersAsync(type, solutionEditor, propertiesToAssign, cancel
                 }
             }
 
-            var lineFormattingOptions = await document
-                .GetLineFormattingOptionsAsync(fallbackOptions: null, cancellationToken).ConfigureAwait(false);
+#if CODE_STYLE
+            var lineFormattingOptions = LineFormattingOptions.Default;
+#else
+            var lineFormattingOptions = await document.GetLineFormattingOptionsAsync(fallbackOptions: null, cancellationToken).ConfigureAwait(false);
```

Comment:
for simplicity can the fallback options just be `LineFormattingOptions.Default` instead of using a preprocessor here?

Reply:
That code is no longer in this PR :)